Fluid.Tests/IncludeStatementTests.cs

```diff
@@ -50,6 +50,8 @@ public class IncludeStatementTests
             {
                 FileProvider = new TestFileProvider("Partials")
             };
+            context.AmbientValues["FluidParser"] = FluidTemplate.Factory.CreateParser();
```

Comment:
Pass the factory only, and create the parser when necessary in the filter

Reply:
Ok, I made this change. Hopefully it is what you are expecting.